pages/api/webhooks/mux.ts

```diff
@@ -11,7 +11,7 @@ const webhookSignatureSecret = process.env.MUX_WEBHOOK_SIGNATURE_SECRET;
 const verifyWebhookSignature = (rawBody: string | Buffer, req: NextApiRequest) => {
   if (webhookSignatureSecret) {
     // this will raise an error if signature is not valid
-    Mux.Webhooks.verifyHeader(rawBody, req.headers['mux-signature'] as string, webhookSignatureSecret);
+    Mux.Webhooks.prototype.verifySignature(Buffer.isBuffer(rawBody) ? rawBody.toString('utf8') : rawBody, req.headers, webhookSignatureSecret)
```

Comment:
Looks like it didn't add

`const mux = new Mux()`

and it went with `Mux.Webhooks.prototype.verifySignature`, which I don't think is right

Reply:
Also interesting that it added this ternary, which wasn't there before:

```
Buffer.isBuffer(rawBody) ? rawBody.toString('utf8') : rawBody, req.headers
```

Reply:
cc @morgante can you take a look at this?

Reply:
```ts
const mux = Mux() // maybe earlier in the file
mux.webhooks.verifyHeader(Buffer.isBuffer(rawBody) ? rawBody.toString('utf8') : rawBody, req.headers, webhookSignatureSecret)
```